ext/RMagick/rmmain.c

```diff
@@ -1295,12 +1295,10 @@ Init_RMagick2(void)
     DEF_ENUM(PixelInterpolateMethod)
         ENUMERATOR(UndefinedInterpolatePixel)
         ENUMERATOR(AverageInterpolatePixel)
-        ENUMERATOR(BicubicInterpolatePixel)
         ENUMERATOR(BilinearInterpolatePixel)
-        ENUMERATOR(FilterInterpolatePixel)
```

Comment:
What happens to `BicubicInterpolatePixel` and `FilterInterpolatePixel`? Are these deprecated constants?

Reply:
This is described in the summary?

Reply:
Oh, sorry, I missed that. Okay, so these are removed. Then these should be part of the 4.0 release, right? Did we have deprecations for them? And for `NearestInterpolatePixel` we should have that as part of 3.2.1?

Reply:
We should have put that in the 3.2.0 release but we missed it. I don't think we need to put out a separate release for this but just do it as a breaking change and release 4.0 after this has been merged.